src/jit/core_templates.expr

```diff
@@ -913,6 +1165,22 @@
 (template: hllboxtype_s
   (^getf (^hllconfig) MVMHLLConfig str_box_type))
 
+(template: hlllist
+  (^getf (^hllconfig) MVMHLLConfig slurpy_array_type))
+
+(template: hllhash
+  (^getf (^hllconfig) MVMHLLConfig slurpy_hash_type))
+
+(template: settypehll!
+  (^setf (^stable $0) MVMSTable hll_owner
+    (call (^func &MVM_hll_get_config_for)
+      (arglist
+        (carg (tc) ptr)
+        (carg $1 ptr)) ptr_sz)))
```

Comment:
I'm a little bit suspicious whether this would come through type specialization... but it looks good.